src/projetos/CadastroProjetosPage.tsx

```
@@ -60,6 +61,14 @@ const formSchema = z.object({
   equipeExecutora: z.array(z.string()).min(1, {
     message: 'Selecione pelo menos um integrante da equipe executora.',
   }),
+  cargaHoraria: z
+    .union([z.number().min(1, { message: 'Carga horária deve ser maior que 0.' }), z.undefined()])
```

ESLint

[error] Replace `z.number().min(1,·{·message:·'Carga·horária·deve·ser·maior·que·0.'·}),·z.undefined()` with `⏎······z.number().min(1,·{·message:·'Carga·horária·deve·ser·maior·que·0.'·}),⏎······z.undefined(),⏎····`

(prettier/prettier)